src/directive/rateLimit.ts

```diff
@@ -104,12 +105,13 @@ export const onLimit: RateLimitOnLimit<Context> = (
       });
     case 'commentOnPost':
     case 'commentOnComment':
-      context.rateLimitCouner.add(1, { type: 'createComment' });
+      context.meter &&
+        counters?.api?.rateLimit?.add(1, { type: 'createComment' });
```

Comment:
We don't need it here anymore right?
```suggestion
      counters?.api?.rateLimit?.add(1, { type: 'createComment' });
```

Reply:
🙈

Reply:
Removed